src/profile/webid.test.ts

```diff
@@ -219,16 +230,19 @@ describe("getProfileAll", () => {
       profileContent
     );
     // Mock the webid unauthenticated fetch
-    const mockedUnauthFetch = jest.requireMock("cross-fetch") as {
+    const mockedUnauthFetch = jest.requireMock("../fetcher") as {
```

Comment:
Can we mock `@inrupt/universal-fetch` instead of `../fetcher`?

Reply:
See the comment about the `fetcher` file